Open-Source-Pundits/nutshell
# got the logo functionality to work!

# Description

Got the logo linked to take you back to the dashboard page.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout JC-logo-functionality
git pull origin JC-logo-functionality
Check to see if the logo will direct you to the dashboard when in a different page.


# Checklist:
